contracts/RMRK/extension/erc20Holder/ERC20Holder.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "./IERC20Holder.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+error InvalidValue();
+error InvalidAddress();
+error InsufficientBalance();
```

Comment:
I'd move this to the Errors library. 🤔

Reply:
I'm more an more against this. This could be a nice stand alone, I feel like we should keep errors for extensions only on extensions :thinking: